index.js

```diff
@@ -982,8 +1022,10 @@ module.exports = function (log, indexesPath) {
           (err, answer) => {
             if (err) cb(err)
             else {
-              debug(
-                `paginate(): ${answer.duration}ms, total messages: ${answer.total}`
+              debugQuery(
+                `paginate(${getNameFromOperation(operation)}): ${
+                  answer.duration
+                }ms, total messages: ${answer.total}`
```

Comment:
Oh, one tiny optimization is to do `debugQuery.enabled && debugQuery(.....)` which means that `getNameFromOperation` would not be called in vain when DEBUG isn't used.